### PR TITLE
Upgrade package json

### LINKED ISSUE #14: Upgrade scripts and packages in package-json file
* Upgrade to avoid vulnerability issues
* Better script use
* Apply formatting after #11

### PR DESCRIPTION
**Pull request by triszt4n**
* Just some simple devops operation to avoid vulnerability messages.
* Newly added script workflow for easier test script writing in `package.json` file.

### CODE REVIEW

**Comment by triszt4n** [comment]
Might be wiser to merge this after #11

**Comment by triszt4n** [comment]
Only ESLint should be failing, see #13

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -26,4 +26,6 @@ jobs:
 
       - name: Install dependencies
         run: npm ci
-      - run: npm run test-cucumber
+        
+      - name: Run tests
+        run: npm run test
```
Comment (triszt4n) [inline .github/workflows/tests.yml:31]: Now this workflow file uses all the test scripts

File: config/db.js [hunk 2]
```diff
@@ -1,13 +1,10 @@
-const mongoose = require('mongoose');
+const mongoose = require('mongoose')
 
-const connectionString = '';
+const connectionString = 'mongodb://localhost:27017/cica'
```
Comment (triszt4n) [inline config/db.js]: This shouldn't be here, let me fix this.

File: package.json [hunk 3]
```diff
@@ -4,27 +4,34 @@
...
-    "test-cucumber": "cucumber-js test/bdd"
+    "test:mocha": "mocha test/unit --recursive",
+    "test:cucumber": "cucumber-js test/bdd",
+    "test:ui": "",
+    "lint:eslint": "eslint --ext .js .",
+    "lint:prettier": "prettier --check .",
+    "lint": "run-s lint:*",
+    "format": "prettier --write .",
+    "test": "run-s test:*",
+    "start": "nodemon index.js"
```
Comment (triszt4n) [inline package.json:15]: Renamed and rearranged the scripts for easier use

File: package.json [hunk 4]
```diff
@@ -4,27 +4,34 @@
...
-    "ejs": "^3.0.1",
-    "express": "^4.17.1",
-    "express-session": "^1.17.0",
-    "moment": "^2.24.0",
-    "mongoose": "^5.9.10"
+    "ejs": "^3.1.8",
+    "express": "^4.18.1",
+    "express-session": "^1.17.3",
+    "moment": "^2.29.3",
+    "mongoose": "^6.3.3"
```
Comment (triszt4n) [inline package.json:24]: Bump versions of dependencies

File: package.json [hunk 5]
```diff
@@ -4,27 +4,34 @@
...
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "mocha": "^7.1.2",
-    "nodemon": "^2.0.2",
-    "prettier": "2.6.2"
+    "prettier": "^2.6.2",
+    "chai": "^4.3.6",
+    "mocha": "^10.0.0",
+    "nodemon": "^2.0.16",
+    "npm-run-all": "^4.1.5"
```
Comment (triszt4n) [inline package.json:35]: Bump versions of devdependencies

**Review by bluebird182 — APPROVED** [review]
Look's good to me.

**Pull request merged**